now.json

```
@@ -89,21 +94,21 @@
       "headers": { "Location": "/docs/v2/introduction" }
     },
     {
-      "src": "/docs/v2/(?:deployments|advanced)/builders/developer-guide(.*)",
+      "src": "/docs/v2/(?:deployments|advanced)/runtimes/developer-guide(.*)",
       "status": 301,
       "headers": {
         "Location": "https://github.com/zeit/now/blob/canary/DEVELOPING_A_BUILDER.md"
       }
     },
     {
-      "src": "/docs/v2/deployments/(?:builders|official-builders)/(.*)/?",
+      "src": "/docs/v2/deployments/(?:runtimes|official-runtimes)/(.*)/?",
```

This actually needs to be builders since it's redirecting from a legacy route.
```suggestion
      "src": "/docs/v2/deployments/(?:builders|official-builders)/(.*)/?",
```